state/pgstatestorage.go

```diff
@@ -1254,6 +1256,32 @@ func (p *PostgresStorage) GetL2BlockHashesSince(ctx context.Context, since time.
 	return blockHashes, nil
 }
 
+// IsBatchConsolidated checks if the batch ID is consolidated
+func (p *PostgresStorage) IsBatchConsolidated(ctx context.Context, batchId int, dbTx pgx.Tx) (bool, error) {
+	q := p.getExecQuerier(dbTx)
+	err := q.QueryRow(ctx, isBatchConsolidated, batchId).Scan(nil)
```

Comment:
I think we should use `q.Query()` for this case instead of `q.QueryRow().Scan(nil)`

see: https://github.com/jackc/pgx/blob/master/conn.go#L538-L548